### ISSUE TITLE
Xamarin Forms

### ISSUE THREAD

**Issue by andersonvieiragomeslopes** [SamDel/ChromeCast-Desktop-Audio-Streamer#21]
Do you work with xamarin.forms? If yes, how? Do you have a manual?

**Comment by SamDel** [comment]
No, it's not in Xamarin. I use [Windows Forms](https://docs.microsoft.com/en-us/dotnet/framework/winforms/) for this application.
